config/prod.exs

```diff
@@ -12,17 +12,19 @@ use Mix.Config
 # manifest is generated by the mix phoenix.digest task
 # which you typically run after static files are built.
 config :remote_retro, RemoteRetro.Endpoint,
+  server: true,
   http: [port: {:system, "PORT"}],
-  url: [scheme: "https", host: System.get_env("HEROKU_HOST"), port: 443],
+  url: [scheme: "https", host: "bar.gigalixirapp.com", port: 443],
   force_ssl: [rewrite_on: [:x_forwarded_proto]],
   cache_static_manifest: "priv/static/manifest.json",
-  secret_key_base: System.get_env("SECRET_KEY_BASE")
+  secret_key_base: "${SECRET_KEY_BASE}"
 
 config :remote_retro, RemoteRetro.Repo,
   adapter: Ecto.Adapters.Postgres,
-  url: System.get_env("DATABASE_URL"),
-  pool_size: String.to_integer(System.get_env("POOL_SIZE") || "10"),
-  ssl: true
+  url: "${DATABASE_URL}",
+  database: "",
+  ssl: true,
+  pool_size: 1

```

Comment:
hey @jesseshieh looking to move to gigalixir, and curious why the poolsize here is 1? Is that due to a limit on the free database provided with Gigalixir. I'm thinking we'll likely keep our Heroku postgres database, and we'd keep the current production connection pool size.

Reply:
You got it. That's the exact reason. If you're not using the gigalixir free database, then feel free to set pool size to whatever you like!